docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+      - .:/var/www/firefly-iii
```

Comment:
The Dockerfile is running it in the folder inside the container after doing a copy from your host folder, therefore the vendor folder is only inside the container, and when you mount your host folder, you override the one inside the container.